auth-web/src/components/auth/staff/review-task/AccountInformation.vue

```diff
@@ -75,7 +75,7 @@ export default class AccountInformation extends Vue {
 
   @Prop({ default: 'Account Information' }) private title: string
   @Prop({ default: {} }) accountUnderReview: any
-  @Prop({ default: {} }) accountUnderReviewAddress: Address
+  @Prop({ default: null }) accountUnderReviewAddress: Address
```

Comment:
why this change?

Reply:
There was this warn message  
![image](https://user-images.githubusercontent.com/78374865/125107842-20816800-e096-11eb-9fe5-39e93607fc59.png)